sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/orc/OrcSourceSuite.scala

```diff
@@ -649,4 +649,14 @@ class OrcSourceSuite extends OrcSuite with SharedSparkSession {
       checkAnswer(spark.sql("SELECT _col0, _col2.c1 FROM t1"), Seq(Row(1, "a")))
     }
   }
+
+  test("SPARK-35612: Support LZ4 compression in ORC data source") {
```

Comment:
Could you move this from `OrcSourceSuite` to `OrcSuite`? Then, `HiveOrcSourceSuite` will test it automatically.

Reply:
It's helpful, thanks